include/internal/common.h

```diff
@@ -186,23 +241,11 @@ static inline int ossl_ends_with_dirsep(const char *path)
     return *path == '/';
 }
 
-static inline char ossl_determine_dirsep(const char *path)
-{
-    if (ossl_ends_with_dirsep(path))
-        return '\0';
-
-# if defined(_WIN32)
-    return '\\';
-# else
-    return '/';
-# endif
-}
-
 static inline int ossl_is_absolute_path(const char *path)
 {
 # if   defined _WIN32
     if (path[0] == '\\'
-        || (path[0] != '\0' && path[1] == ':'))
+            || (path[0] != '\0' && path[1] == ':'))
```

Comment:
This could just go on asingle line, no? Also, I don't know much about Windows but can really any non-NUL character be used here?

Reply:
Well, any letter is okay (network drives via `net use` command can pick a letter).  But if someone enters `?:\foo.conf` as a config file, then they get whatever they desrve; not worth the `isupper` check.